packages/mdc-base/component.ts

```diff
@@ -114,4 +114,4 @@ class MDCComponent<FoundationType extends MDCFoundation> {
   }
 }
 
-export default MDCComponent;
+export {MDCComponent as default, MDCComponent};
```

Comment:
just curious why you decided to export both ways.

Reply:
This way it doesn't break people using the default export, but it allows them to use the named one if they want. At some point you could deprecate the default one and eventually remove it.

Reply:
Ok, if we do this then we should be doing this across all components then, correct?

Reply:
If eliminating default exports is something you want to do eventually, then yeah I would do this kind of thing to all the ones that have default exports